guides/common/modules/con_limitations-of-installation-in-an-ipv6-network.adoc

```diff
@@ -7,3 +7,8 @@
 
 * Although {Project} provisioning templates include IPv6 support for PXE and HTTP (iPXE) provisioning, the only tested and certified provisioning workflow is the UEFI HTTP Boot provisioning.
 This limitation only relates to users who plan to use {Project} to provision hosts.
+
+[NOTE]
+====
+In {ProjectVersion}, virt-who does not support IPv6.
+====
```

Comment:
The whole module is a list of limitations, in the format of a bulleted list. When adding a new limitation, it's better to work with the existing format (for consistency) rather than start introducing a different format (notes).

Also, there is no need to specify the current version explicitly. The whole guide is published specifically for this version.

Therefore:

```suggestion
* virt-who does not support IPv6.
```

That would, however, mean starting a sentence with a lowercase character, so perhaps this instead:

```suggestion
* {Project} does not support virt-who in an IPv6 network.
```